src/java.base/share/classes/java/math/MathContext.java

```diff
@@ -102,8 +102,8 @@ public final class MathContext implements Serializable {
      * A {@code MathContext} object with a precision setting
      * matching the precision of the IEEE 754-2019 decimal128 format, 34 digits, and a
      * rounding mode of {@link RoundingMode#HALF_EVEN HALF_EVEN}.
-     * Note the exponent range of decimal64 is <em>not</em> used for
-     * rounding.
+     * Note the exponent range of decimal128 (min exponent of -6143,
+     * max exponent of 6144) is <em>not</em> used for rounding.
```

Comment:
The exponent ranges are all correct, but... why mention them in the first place if they are not used?